src/Illuminate/Http/Client/RequestException.php

```diff
@@ -26,6 +33,37 @@ public function __construct(Response $response)
         $this->response = $response;
     }
 
+    /**
+     * Enable truncation of the exception message.
+     *
+     * @return void
+     */
+    public static function truncate()
+    {
+        static::$truncateAt = 120;
+    }
```

Comment:
Isn't that a little redundant as you already set the property to this value by default? Or is that for resetting purposes? If it is needed, I personally don't like magic numbers being set on properties with no indication why they are the default

Reply:
Yea this is to reset the value back to the default value.